features/admin.roles.v2/components/edit-role/edit-role-users.tsx

```diff
@@ -168,6 +177,7 @@ export const RoleUsersList: FunctionComponent<RoleUsersPropsInterface> = (
 
         const alreadyAssignedUsersFromSelectedUserStore: UserBasicInterface[] = role?.users?.map(
             (user: RolesMemberInterface) => ({
+                displayName: user.displayName,
```

Comment:
Typecheck is failing due the following error. Can you check this ?

```
 Error: ../../features/admin.roles.v2/components/edit-role/edit-role-users.tsx(180,35): error TS2339: Property 'displayName' does not exist on type 'RolesMemberInterface'.
```

Reply:
Fixed with [8d36af2](https://github.com/wso2/identity-apps/pull/8511/commits/8d36af28ecd77948681a037ed26ff739beb73c7d)